server/routes.js

```diff
@@ -0,0 +1,8 @@
+const routes = function (app, postManager, userManager) {
+	app.post('/add_post', function (req, res) {
+		console.log(req.body);
+		res.redirect('/');
```

Comment:
This redirect should be managed by the client (see the routing comment above).
This important because your server shouldn't know about the business flow, pages, and so on. You also might have multiple places to add a post (for instance, an additional pop-up from a completely different page).